aspnetcore/mvc/views/tag-helpers/built-in/partial-tag-helper.md

```diff
@@ -75,7 +75,17 @@ In the preceding code, the `IsNumberReadOnly` key value is set to `true` and add
 
 In this example, the value of `ViewData["IsNumberReadOnly"]` determines whether the *Number* field is displayed as read only.
 
+## Migrate from an HTML Helper
+
+Consider the following asynchronous HTML Helper example. A collection of products is iterated and displayed. Per the `PartialAsync` method's first parameter, the *_ProductPartial.cshtml* partial view is loaded. An instance of the `Product` model is passed to it for binding.
```

Comment:
This isn't necessary to change, but I've personally been trying to steer clear of pronoun use. You can replace "it" with "the view" if you feel the same way ...

> An instance of the \`Product\` model is passed to the view for binding.

Side point: I'm sure ur aware that the language is very *view-ish* in light of RP.

Reply:
It's intentionally heavy on the word "view", as that's the purpose of this Tag Helper.